styles/styles.css

```diff
@@ -68,6 +68,8 @@
   --navigation-dropdown-background-color: #f2efef;
   --navigation-dropdown-border-color: #dcdadb;
   --navigation-dropdown-hover-color: #f8f8f8;
+  --progress-bar-greyed-color: #9d9d9c;
```

Comment:
don't use actual color names in variables, but e.g. dark/light or primary/secondary, etc.